src/passes/Inlining.cpp

```diff
@@ -540,91 +540,15 @@ struct FunctionSplitter {
   // the number of iterations, etc.). Therefore this function will only find out
   // if we *can* split, but not actually do any splitting.
   bool canSplit(Function* func) {
+    // TODO: we should not get here if function marked uninlineable.
```

Comment:
What did you have in mind here?

(if you mean the existing `uninlineable` field then that is only used for `canHandleParams`, which is checked below us)

Reply:
(lgtm % this question)

Reply:
I was thinking to have worthInlining to do the checking for that (had the follow up in here: cl/527177213)

Reply:
I see. That sounds ok, but can you remove the comment from this PR then? (If it only makes sense in the context of the later PR, then it makes more sense there, and removing it here will avoid confusion in reading the code in the future e.g. in bisection).